client/src/call-customer/components/board.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useRef } from 'react'
+import '../call-customer.css';
+import Clock from './clock.tsx';
+import { BsFillPeopleFill } from "react-icons/bs";
+import { BsFillPersonFill } from "react-icons/bs";
+import { TbHourglassEmpty } from "react-icons/tb";
+
+function DisplayBoard() {
+
+    const [ticketNo, setTicketNo] = useState(110);
+    const queue1: number = 0;
+    const queue2: number = 1;
+    const queue3: number = 5;
+
+
+    return (
+        <>
+            <div className='full-page'>
+                <div className='column'>    
+                    <Clock />     
+                    {/* Ticket Number Box */}
+                    <div className='ticket-box'>
+                        <span className='ticket-text'>
+                            Ticket No
+                        </span>
+                        <br/>
+                        <span className='ticket-number'>
+                            {ticketNo}
+                        </span>
+                        <br/>
+                        <span className='ticket-subtext'>
+                            please proceed to
+                        </span>
+                        <br/>
+                        <span className='ticket-counter'>
+                            Counter 01
+                        </span>                
+                    </div>
+                </div>
+                <div className='column'>
+                    {/* Waiting Queue Box */}
+                    <div className='queue-title'>
+                        <span className='queue-text'>
+                            Waiting Queue
+                        </span>
+                    </div>
+                    <div className='queue-box'>
+                        <div className='service-queue'>
+                            <span className='service-queue-lentgh'>{queue1}</span>
+                            <span className='service-queue-icon'>
+                                {queue1 === 0 ? <TbHourglassEmpty /> : queue1 === 1 ? <BsFillPersonFill /> : <BsFillPeopleFill />}
```

Comment:
I have some doubts about showing the 0 length queues or not.
Please share your idea with me. Much appreciated.

Reply:
Yeah you're right, we can avoid showing the 0 length queues. They only occupy space in the page, which is already small